tests/phpunit/tests/option/autoloadValue.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * @group option
+ *
+ */
+class Tests_Option_Autoload_Value extends WP_UnitTestCase {
+
+	/**
+	 * @covers ::get_autoload_value
+	 * @ticket 42441
+	 *
+	 * @dataProvider data_values
+	 *
+	 * @param $autoload
+	 * @param $expected
+	 * @return void
```

Comment:
```suggestion
	 * @ticket 42441
	 *
	 * @covers ::get_autoload_value
	 *
	 * @dataProvider data_values
	 *
	 * @param $autoload
	 * @param $expected
```

Per [56070](https://core.trac.wordpress.org/ticket/56070) proposal update the annotations and remove `@return void`